acs-core/src/main/java/com/arrow/acs/client/model/CompanyModel.java

```diff
@@ -23,6 +26,7 @@
 	private PasswordPolicyModel passwordPolicy;
 	private LoginPolicyModel loginPolicy;
 	private String parentCompanyHid;
+	private Set<String>   impersonateUsers = new HashSet<>();;
```

Comment:
please format this line - a lot of spaces before variable name
please remove last ";"